src/controllers/check.your.answers.controller.ts

```diff
@@ -1,16 +1,25 @@
+import { Session } from "ch-node-session-handler";
 import { NextFunction, Request, Response } from "express";
 import ObjectionCompanyProfile from "../model/objection.company.profile";
 import { Templates } from "../model/template.paths";
+import { Objection } from "../modules/sdk/objections";
+import { getObjection } from "../services/objection.service";
 import { retrieveCompanyProfileFromObjectionSession } from "../services/objection.session.service";
 import logger from "../utils/logger";
 
-export const get = async (req: Request, res: Response, next: NextFunction) => {
+export const get = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
   if (req.session) {
     try {
       const company: ObjectionCompanyProfile = retrieveCompanyProfileFromObjectionSession(req.session);
```

Comment:
I know it's not your change but a trick you can use if you just want values from an object (like company)
const {companyName, companyNumber} = retrieveCompanyProfileFromObjectionSession(req.session);

then you can add them to res.render just as 
companyName,
companyNumber,
objection, etc..

Reply:
you may need to use 'as ObjectionCompanyProfile' on teh end

Reply:
👍  will implement.